### PR TITLE
Slowed down heartbeat times to lighten load on hibike python stack.

### PR DESCRIPTION
**Pull request by dHutchings**
Slowed to every 200 ms, and timeout after 1 sec.
Was every 100ms, timeout after 0.5 sec.

### CODE REVIEW

**Comment by dHutchings** [comment]
Added new functionality so that there's a default timeout.

Tested to work in both default & non-default modes.

File: hibike/lib/hibike/hibike_device.cpp [hunk 1]
```diff
@@ -2,19 +2,27 @@
 
 message_t hibikeBuff;
 uint16_t params, old_params, subDelay;
-uint32_t value, disable_latency, heartbeatDelay, prevHeartTimer;
-uint64_t prevSubTime, currTime, sent_heartbeat, resp_heartbeat;
+
+//Heartbeat settings.  
+uint32_t disable_latency, heartbeat_Delay;
+uint32_t default_disable_latency = 1000; //number of ms without heartbeat untill disable.  default
+uint32_t default_heartbeat_Delay = 200; //default time between sending heartbeat requests.
```
Comment (nikitakit) [inline hibike/lib/hibike/hibike_device.cpp]: I think you can should mark these as `const`

File: hibike/lib/hibike/hibike_device.cpp [hunk 2]
```diff
@@ -2,19 +2,27 @@
 
 message_t hibikeBuff;
 uint16_t params, old_params, subDelay;
-uint32_t value, disable_latency, heartbeatDelay, prevHeartTimer;
-uint64_t prevSubTime, currTime, sent_heartbeat, resp_heartbeat;
+
+//Heartbeat settings.  
+uint32_t disable_latency, heartbeat_Delay;
```
Comment (nikitakit) [inline hibike/lib/hibike/hibike_device.cpp]: if you're going to be renaming the variables, could you standardize the capitalization in `heartbeat_Delay`. Either camelCase or snake_case is fine, but a mix seems odd.

**Review by nikitakit — CHANGES_REQUESTED** [review]

File: hibike/lib/hibike/hibike_device.cpp (hunk 1 above)
Comment (dHutchings) [reply hibike/lib/hibike/hibike_device.cpp]: done.

File: hibike/lib/hibike/hibike_device.cpp (hunk 2 above)
Comment (dHutchings) [reply hibike/lib/hibike/hibike_device.cpp]: done.

**Review by nikitakit — APPROVED** [review]
diff lgtm (but please makes sure that this compiles and runs on hardware)

**Comment by dHutchings** [comment]
It does.  Thank you!

**Pull request merged**